internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -263,6 +263,44 @@ func (vc *WSVersionCompatibility) processConfigTableUpdates(uncompressedPayload
 	return processedPayload, nil
 }
 
+func (vc *WSVersionCompatibility) removeCoreEntity(
+	processedPayload string,
+	entityType string,
+	dataPlaneVersionStr string,
+	configTableKey string,
+	changeID ChangeID,
+	tracker *ChangeTracker,
+) string {
+	entity := gjson.Get(processedPayload, fmt.Sprintf("config_table.%ss", entityType))
```

Comment:
```suggestion
	entity := gjson.Get(processedPayload, fmt.Sprintf("config_table.%s", entityType))
```

Reply:
The testing is in #464 using entity deletion rules for the Vaults.  I'd rather do the testing in this PR, but it felt silly to write mock entity deletion rules for an entity that we will never want to delete.

Reply:
I moved the testing into this PR.